### PR TITLE
Update microsoft-defender-antivirus-on-windows-server.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated another cmdlet sc query state= all to view non-running services.

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7334

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-on-windows-server.md [hunk 1]
```diff
@@ -121,6 +121,12 @@ sc query Windefend
 
 The `sc query` command returns information about the Microsoft Defender Antivirus service. When Microsoft Defender Antivirus is running, the `STATE` value displays `RUNNING`.
 
+To view all the Non running services, run the following Powershell cmdlet:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-on-windows-server.md]:
```suggestion
To view all the services that are not running, run the following Powershell cmdlet:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**